athena-federation-sdk/README.md

```diff
@@ -102,6 +102,17 @@ public class MyMetadataHandler extends MetadataHandler
     protected ListTablesResponse doListTables(BlockAllocator blockAllocator, ListTablesRequest request)
     {
       //Return a list of tables (strings) for the requested catalog and schema
+      //The request (ListTablesRequest) includes a nextToken (String) and pageSize (int) attributes that can
+      //be used to paginate the response. The nextToken (request.getNextToken()) will be null if this is the first
+      //paginated request, or will contain a value of the starting point for the current request. pageSize (request
+      //.getPageSize()) will be -1 (ListTablesRequest.UNLIMITED_PAGE_SIZE_VALUE) for non-paginated requests (full
+      //list of tables should be returned), or it will be an integer value > 0. The latter indicates the maximum
+      //number of tables to return in the response (ListTablesResponse).
+      //The response (ListTablesResponse) also has a nextToken (String) attribute used for pagination. Its value
+      //should be set with the starting point for the next paginated request. If all tables have been processed, the
+      //value returned for nextToken should be null.
+      //It is noteworthy to mention that the nextToken value should be obfuscated by the connector implementing the
+      //pagination logic via some encryption algorithm.
```

Comment:
this seems like too much detail, maybe just change the existing sentence to "Return a paginated list of tables....". This looks like a copy paste from the README so i can understand why you did it, but its just too much info in my opinion.

Reply:
Reduced.